### PR TITLE
Fix issue in CKV_K8S_28 and CKV_K8S_37

### PR DESCRIPTION
**Pull request by VimukthiPerera**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

Fixes https://github.com/bridgecrewio/checkov/issues/1021

### CODE REVIEW

File: checkov/kubernetes/checks/DropCapabilities.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def scan_spec_conf(self, conf):
             if conf["securityContext"].get("capabilities"):
                 if conf["securityContext"]["capabilities"].get("drop"):
                     for d in conf["securityContext"]["capabilities"]["drop"]:
-                        if "ALL" in d or "NET_RAW" in d:
+                        if "ALL" in d or "all" in d or "NET_RAW" in d:
```
Comment (gruebel) [inline checkov/kubernetes/checks/DropCapabilities.py:26]: perfect use case for `any()`
```suggestion
                        if any(cap in d for cap in ("ALL", "all", "NET_RAW")):
```

File: checkov/kubernetes/checks/MinimizeCapabilities.py [hunk 2]
```diff
@@ -21,7 +21,7 @@ def scan_spec_conf(self, conf):
             if conf["securityContext"].get("capabilities"):
                 if conf["securityContext"]["capabilities"].get("drop"):
                     for d in conf["securityContext"]["capabilities"]["drop"]:
-                        if "ALL" in d:
+                        if "ALL" in d or "all" in d:
```
Comment (gruebel) [inline checkov/kubernetes/checks/MinimizeCapabilities.py:24]:
```suggestion
                        if any(cap in d for cap in ("ALL", "all")):
```

**Review by gruebel — COMMENTED** [review]
And it would be great, when you also add tests for those changes :)

**Review by nimrodkor — APPROVED** [review]

**Comment by nimrodkor** [comment]
Thanks @VimukthiPerera & @gruebel !

**Pull request merged**